src/components/MovieOverview.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import '../styles/MovieOverview.css'
 
 class MovieOverview extends React.Component {
```

Comment:
Just curious as to you reasoning for making this a class based component? If it doesn't have state, it can probably just be functional.